src/react-components/footer.js

```diff
@@ -0,0 +1,89 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import MobileDetect from "mobile-detect";
+import FontAwesomeIcon from "@fortawesome/react-fontawesome";
+import faUsers from "@fortawesome/fontawesome-free-solid/faUsers";
+import faEllipsisH from "@fortawesome/fontawesome-free-solid/faEllipsisH";
+import faShareAlt from "@fortawesome/fontawesome-free-solid/faShareAlt";
+import faExclamation from "@fortawesome/fontawesome-free-solid/faExclamation";
+import faTimes from "@fortawesome/fontawesome-free-solid/faTimes";
+
+import styles from "../assets/stylesheets/footer.scss";
+
+const mobiledetect = new MobileDetect(navigator.userAgent);
+
+export default class Footer extends Component {
+  static propTypes = {
+    hubName: PropTypes.string,
+    participantCount: PropTypes.number,
+    onClickInvite: PropTypes.func,
+    onClickReport: PropTypes.func
+  };
+  state = {
+    menuVisible: false
+  };
+  render() {
+    const menuVisible = this.state.menuVisible;
+    return (
+      <div className={styles.container}>
+        {mobiledetect.mobile() ? (
```

Comment:
I think we should change this to use media queries based upon screen width, sorry I wasn't more clear in the mockup